src/js/components/form-elements/v2/DateField.jsx

```diff
@@ -28,6 +29,10 @@ const DateField = ({
   showTimeSelect,
   hideErrorMessageWrapper,
   customDateFormat,
+  fieldIndex,
+  fieldId,
+  focusIndex,
+  focusId,
```

Comment:
You can pass those as, I don't know, `focusProps`? `matchProps`? Instead of having 4 properties separatelly